package.json

```diff
@@ -16,23 +16,26 @@
     "test"    : "jest"
   },
   "devDependencies" : {
-    "react-tools" : "0.12.2",
-    "jest-cli"    : "0.1.17",
-
+    "6to5-core"                   : "^3.6.5",
```

Comment:
Looks like we're locking things down more strictly. None of the other dependencies have `^` or `~` or `*` or anything like that.